pkg/data/mobileService.go

```diff
@@ -22,26 +51,67 @@ func (sa *secretAttributer) GetName() string {
 	return strings.TrimSpace(string(sa.Secret.Data["name"]))
 }
 
+// MobileServiceRepo implments the mobile.ServiceCruder interface. it backed by the secret resource in kubernetes
+type MobileServiceRepo struct {
+	client     corev1.SecretInterface
+	convertors map[string]SecretConvertor
+	logger     *logrus.Logger
+}
+
+// NewMobileServiceRepo returns a new MobileServiceRepo
 func NewMobileServiceRepo(client corev1.SecretInterface) *MobileServiceRepo {
 	return &MobileServiceRepo{
 		client: client,
+		convertors: map[string]SecretConvertor{
+			"fh-sync-server": defaultSecretConvertor{},
```

Comment:
would it be better to have specific-convertors only here, and anything else just uses the default, rather than having to list out every service that uses the default convertor.

Reply:
updated to try using the default convertor if non found in the map. I originally went this route as I think it is more obvious what is going on.